### PR TITLE
Add WeChatComponentAPI and WeChatComponentClient

### PR DESCRIPTION
**Pull request by hunter007**
实现公众号开放平台代公众号调用接口

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request
- Commit: 12b9e0122e66ebba48333dee200cd6ee91ff9a5a
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**
- Commit: c8b5f773c622ffc3a91b8f56930d1a253210c431
  - Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/jxtech/wechatpy/blob/master/CONTRIBUTING.md

---

This message was auto-generated by https://gitcop.com

**Comment by messense** [comment]
Thanks.

能解决下冲突吗？

File: wechatpy/client/__init__.py [hunk 1]
```diff
@@ -1,11 +1,78 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, unicode_literals
-
+import time
 from wechatpy.client.base import BaseWeChatClient
 from wechatpy.client import api
 
 
+class WeChatComponentClient(BaseWeChatClient):
```
Comment (messense) [inline wechatpy/client/__init__.py:8]: 这里也许继承 WeChatClient 更好？下面的 API endpoints 就可以不写了，否则以后增加 endpoints 两处都要增加。

**Comment by hunter007** [comment]
把这个 request删除吧，我合并之后在提交

File: wechatpy/client/__init__.py (hunk 1 above)
Comment (hunter007) [inline wechatpy/client/__init__.py:8]: 我考虑过扩充WeChatClient，但这样会改变WeChatClient的初始化参数。所以直接写了两个类。

我等下去继承一下看怎么样。

**Pull request closed without merging**